src/main/java/com/scopely/mapper/JsonDynamoMapper.java

```diff
@@ -66,15 +70,19 @@
  */
 @SuppressWarnings({"WeakerAccess", "unused"})
 public class JsonDynamoMapper {
+    private final int BATCH_SAVE_UNPROCESSED_MAX_TRIES = 20;
+
     private final AmazonDynamoDB amazonDynamoDB;
+    private final DynamoDB dynamoDB;
     private final ObjectMapper objectMapper;
 
-    public JsonDynamoMapper(AmazonDynamoDB amazonDynamoDB) {
-        this(amazonDynamoDB, new ObjectMapper().registerModule(new Jdk8Module()));
+    public JsonDynamoMapper(AmazonDynamoDB amazonDynamoDB, DynamoDB dynamoDb) {
```

Comment:
There 2 other alternatives to forcing the new parameter:
1. Create it inside this constructor
2. Leave it as `null` and throw an exception if it's used

Reply:
`AmazonDynamoDB` has a [`batchWriteItem`](https://www.javadoc.io/doc/com.amazonaws/aws-java-sdk-dynamodb/1.10.66/com/amazonaws/services/dynamodbv2/AmazonDynamoDB.html#batchWriteItem(com.amazonaws.services.dynamodbv2.model.BatchWriteItemRequest)) method, can we use that for batch write support?

I think we should avoid introducing a new dependency to this class if possible.

Reply:
Oh you're right, I changed it to using `AmazonDynamoDB`. Thank you!